supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -981,6 +981,17 @@ With a _cross-forest trust_ between an Active Directory (AD) forest root domain
 
 *See also*:
 
+[[currently]]
+==== image:images/yes.png[yes] currently (adverb)
+*Description*: Use _currently_ to refer to a situation that will change in the future. For example, you can use _currently_ in release notes when a known issue will likely be fixed in an upcoming release. 
```

Comment:
* Since we're telling how to use it, then it should be quotes
* Also softened "will change" to "might change", since I think it can still be currently even if it doesn't end up changing.

```suggestion
*Description*: Use "currently" to refer to a situation that might change in the future. For example, you can use "currently" in release notes when a known issue will likely be fixed in an upcoming release. 
```